integration-test/rules/oversized-1.yml

```diff
@@ -0,0 +1,517 @@
+title: Query Splitting Test Rule
+id: 26654045-413c-4db1-be5f-36f34733a865
+status: test
+description: Tests that query splitting is working correctly in Loki.
+author: jamesc-grafana
+date: 2025-09-25
+logsource:
+  product: github
+  service: audit
+detection:
+  selection:
+    username:
+      - "admin0"
```

Comment:
Hah, that's definitely one way of testing this! You can also have rules with a list of conditions- it's a slightly smaller way of generating multiple queries ☺️